lib/zoo.rb

```
@@ -0,0 +1,64 @@
+require_relative 'animal'
+
+class Zoo
+  attr_reader :name, :street, :city, :state, :zip_code, :inventory
+
+  def initialize(name='City Zoo', street='123 Main St', city='San Juan', state='PR', zip_code='00901')
```

be careful setting too many default values here, since we HAVE to have these details for the address later, it's best to let the code throw an error if a user doesn't provide a necessary detail